aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/NetworkUtilsSteps.java

```diff
@@ -77,6 +78,32 @@ public void setDeviceMqtt(final boolean connectivity) throws IOException, Interr
         }
     }
 
+    /**
+     * Add loop back address.
+     *
+     * @param address the value of loop back address
+     * @throws IOException on IO errors
+     * @throws InterruptedException when thread has been interrupted
+     */
+    @Then("I add loop back address {string}")
+    public void addLoopBackAddress(final String address) throws IOException, InterruptedException {
+        LOGGER.info("Adding loopback address {}", address);
+        platform.networkUtils().addLoopbackAddress(address);
+    }
+
+    /**
+     * Delete loop back address.
+     *
+     * @param address the value of loop back address
+     * @throws IOException on IO errors
+     * @throws InterruptedException when thread has been interrupted
+     */
+    @Then("I remove loop back address {string}")
```

Comment:
```suggestion
     I remove IP address {string} from loop back interface
```

Reply:
fixed in b0f3f75a79ffe49f7ffbb15e0ea553cc9deb6a7a